themes/THEMES.md

```diff
@@ -287,3 +287,7 @@
 ## `zork`
 
 [![](zork/zork-dark.png)](zork/zork-dark.png)
+
+## `lambda`
+
+![](lambda/lambda-dark.png)
```

Comment:
Thank you for updating.

The themes should be ordered in alphabetical order. Can you move this?

Also, the link to the image file should be added for consistency with the other themes.
```suggestion
[![](lambda/lambda-dark.png)](lambda/lambda-dark.png)
```